front/src/routes/ApplicantsPage.svelte

```diff
@@ -78,6 +93,33 @@
     }));
     loading = false;
   });
+
+  let selectedCommittee;
+
+  const passesCommitteeFilters = (
+    rows: RowEntity[],
+    selectedCommittee: CommitteeType
```

Comment:
I just noticed it but are you passing in the `selectedCommittee` variable, which is already defined above? If so, you can just directly use it and you don't have to pass it. Just make sure that functionality still works cause svelte is precompiled and that might break the reactivity

Reply:
this does break reactivity when removed--should I leave as is or try to get around it?

Reply:
(I tried to implement w/ $: block but it also needed both rows and selectedCommittee as inputs)

Reply:
I see. I was thinking that it would happen, I have a solution with `$` in mind, but I don't think it will be in any way clearer or more elegant, so let's just leave it the way you wrote it. It's good solution